### DISCUSSION TITLE
Log Viewer Assets Path Configuration

### DISCUSSION THREAD

**Discussion by PratanuKhajanchi123** [opcodesio/log-viewer#315] [Q&A]
I want to configure the assets path in public folder. Right now the assets created under vendor folder
for reference
![image](https://github.com/opcodesio/log-viewer/assets/156180385/3da9c860-880c-4a02-b840-d5e281cee20f)
Now i want move these assets say /public/folder1/folder2/ path, how can i do it?

**Comment by arukompas** [comment]
Hey @PratanuKhajanchi123 ,

Hm, this isn't something that's supported out of the box, but you could try this hacky way:

1. Move the vendor files somewhere else
2. Publish the Log Viewer layout file: `php artisan vendor:publish --tag=log-viewer-views`
3. Update the layout, changing the paths to `app.js` and `app.css` and the `log-viewer-32.png` icon
4. Update the `mix-manifest.json` contents

Let me know if this works out :)
